start

karvalolka opened the pull request:
+ авторизация;
+ bootstrap;
+ БД;
+ миграции;

karvalolka committed:
+ авторизация;
+ bootstrap;
+ БД;
+ миграции;

kovdmit commented on database/migrations/2024_09_03_181523_create_chars_table.php:
```diff
@@ -0,0 +1,33 @@
+        Schema::create('char', function (Blueprint $table) {
+            $table->id();
+            $table->string('nickname');
+            $table->unsignedTinyInteger('уровень');
```
Шо за прикол с русским названием колонок?)

kovdmit commented on database/migrations/2024_09_03_181523_create_chars_table.php:
```diff
@@ -0,0 +1,33 @@
+            $table->id();
+            $table->string('nickname');
+            $table->unsignedTinyInteger('уровень');
+            $table->unsignedMediumInteger('ХП');
```
Поля опыт нет

kovdmit commented on database/migrations/2024_09_03_184334_create_inventories_table.php:
```diff
@@ -0,0 +1,29 @@
+        Schema::create('inventories', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('золото');
+            $table->text('лут');
```
Русское поле

kovdmit commented on database/migrations/2024_09_03_184334_create_inventories_table.php:
```diff
@@ -0,0 +1,29 @@
+            $table->id();
+            $table->unsignedInteger('золото');
+            $table->text('лут');
+            $table->timestamps();
```
Нет связи с персонажем

kovdmit commented on database/migrations/2024_09_03_184030_create_weapons_table.php:
```diff
@@ -0,0 +1,29 @@
+            $table->id();
+            $table->string('name');
+            $table->unsignedSmallInteger('атака');
+            $table->timestamps();
```
Как определять тип оружия?

kovdmit commented on database/migrations/2024_09_03_184030_create_weapons_table.php:
```diff
@@ -0,0 +1,29 @@
+        Schema::create('weapons', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->unsignedSmallInteger('атака');
```
Русское поле

kovdmit commented on database/migrations/2024_09_03_183919_create_armors_table.php:
```diff
@@ -0,0 +1,29 @@
+            $table->id();
+            $table->string('name');
+            $table->unsignedSmallInteger('защита');
+            $table->timestamps();
```
Как определять тип брони?

kovdmit commented on database/migrations/2024_09_03_183919_create_armors_table.php:
```diff
@@ -0,0 +1,29 @@
+        Schema::create('armors', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->unsignedSmallInteger('защита');
```
Русское поле

kovdmit commented on database/migrations/2024_09_03_182022_create_races_table.php:
```diff
@@ -0,0 +1,28 @@
+        Schema::create('races', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->timestamps();
```
Нет связи со фракцией

kovdmit commented on database/migrations/2024_09_03_181850_create_classes_table.php:
```diff
@@ -0,0 +1,28 @@
+        Schema::create('classes', function (Blueprint $table) {
+            $table->id();
+            $table->string('nameClasses');
+            $table->timestamps();
```
Нет связи с расой

kovdmit commented on database/migrations/2024_09_03_181850_create_classes_table.php:
```diff
@@ -0,0 +1,28 @@
+    {
+        Schema::create('classes', function (Blueprint $table) {
+            $table->id();
+            $table->string('nameClasses');
```
Поля в бд принято именовать в snake_case

karvalolka committed:
исправлен нейминг колонок в БД (ru->eng);
в tab char + поле xp;
изправлена ошибка в формате нейминга поля в БД (snake_case);
+ привязка (char+inventary, race+fraction, (protection_areas+armor/weapon);
+ модели к миграциям
переименнованы модель и миграция (class->grade)
+ модель и миграция (protection_areas)
